runtime/gc_realtime/RealtimeAccessBarrier.cpp

```diff
@@ -950,5 +950,14 @@ MM_RealtimeAccessBarrier::forwardReferenceArrayCopyIndex(J9VMThread *vmThread, J
 	return -2;
 }
 
+void
+MM_RealtimeAccessBarrier::preMountContinuation(J9VMThread *vmThread, j9object_t contObject)
+{
+	MM_EnvironmentBase *env = MM_EnvironmentBase::getEnvironment(vmThread->omrVMThread);
```

Comment:
convert it to MM_EnvironmentRealtime right here and then you don't need a cast later